libs/top_level_metrics.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import pandas as pd
+
+from covidactnow.datapublic.common_fields import CommonFields
+from libs.datasets import combined_datasets
+from libs.datasets.timeseries import TimeseriesDataset
+
+
+def calculate_top_level_metrics_for_fips(fips: str):
+    timeseries = combined_datasets.load_us_timeseries_dataset()
+    latest = combined_datasets.load_us_latest_dataset()
+
+    fips_timeseries = timeseries.get_subset(fips=fips)
+    fips_record = latest.get_record_for_fips(fips)
+
+    # not sure of return type for now, could be a dictionary, or maybe it would be more effective
+    # as a pandas dataframe with a column for each metric.
+    return calculate_top_level_metrics_for_timeseries(fips_timeseries, fips_record)
+
+
+def calculate_top_level_metrics_for_timeseries(timeseries: TimeseriesDataset, latest: dict):
+    # Making sure that the timeseries object passed in is only for one fips.
+    assert len(timeseries.all_fips) == 1
+    population = latest[CommonFields.POPULATION]
+    neg_tests_cumulative = timeseries.data[CommonFields.NEGATIVE_TESTS]
+    neg_tests_daily = neg_tests_cumulative.diff()
+
+    pos_tests_cumulative = timeseries.data[CommonFields.POSITIVE_TESTS]
+    pos_tests_daily = pos_tests_cumulative.diff()
+
+    case_density = calculate_case_density(cases=pos_tests_daily, population=population)
+    test_positivity = calculate_test_positivity(
+        pos_cases=pos_tests_daily, neg_tests=neg_tests_daily
+    )
+
+    return pd.DataFrame({"caseDensity": case_density, "testPositivity": test_positivity}).replace(
```

Comment:
Can you include the date index with this result? Also, I generally try to include fips in any output data frames as well, makes it much easier to join and combine multiple regions if desired